src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionInfo.cs

```diff
@@ -57,6 +57,13 @@ public ConnectionInfo(ISqlConnectionFactory factory, string ownerUri, Connection
         internal readonly ConcurrentDictionary<string, DbConnection> ConnectionTypeToConnectionMap =
             new ConcurrentDictionary<string, DbConnection>();
 
+        public ConcurrentDictionary<string, DbConnection> ConnectionMap
+        {
+            get {
```

Comment:
`{` on separate line in C#...I won't comment on any more of these, but please fix them all